src/utilities/dataLayer.ts

```diff
@@ -11,5 +11,5 @@ export function pushTrackingEvent(
     context?: Serializable
 ): void {
     window.dataLayer = window.dataLayer || [];
-    window.dataLayer.push({ event, action, context });
+    window.dataLayer.push({ event, action, context, _clear: true });
```

Comment:
Is er geen situatie denkbaar waar we clear op false willen zetten ? Of is dat zo uitzonderlijk dat je dan maar zelf window.dataLayer moet aanroepen en dat we nu de helper niet te complex willen maken.

Reply:
@martinbroos ik de functie aangepast dat default de _clear op true staat en false mee kan geven indien nodig.

Wat vind je daar van?